index.js

```diff
@@ -36,45 +43,51 @@ const findIn = (root, ...tags) => {
 	return el
 }
 
-const parsePosList = (_, transformCoords, stride = 2) => {
+const createChildContext = (_, opts, ctx) => {
+	const srsDimensionAttribute = _.attributes && _.attributes.srsDimension
+	if (srsDimensionAttribute) {
+		const srsDimension = parseInt(srsDimensionAttribute)
+		if (Number.isNaN(srsDimension) || srsDimension <= 0) throw new Error(`invalid srsDimension attribute value "${srsDimensionAttribute}", expected a positive integer`)
```

Comment:
Please try to stay below 80 non-space characters per line, as long as that doesn't require ugly workarounds. In this case would IMO more readable:

```js
if (Number.isNaN(srsDimension) || srsDimension <= 0) {
	throw new Error(`invalid srsDimension attribute value "${srsDimensionAttribute}", expected a positive integer`)
}
```